data_steward/analytics/cdr_ops/controlled_tier_qc/code/config.py

```diff
@@ -1,12 +1,15 @@
 from pathlib import Path
 
-CSV_FOLDER = Path('csv')
-SQL_FOLDER = Path('sql')
+CSV_FOLDER = Path('analytics/cdr_ops/controlled_tier_qc/CSV')
```

Comment:
Should this be `csv`?

Reply:
you are right it should be 'csv'. I'll update it.

Reply:
And yes, other changes were due to YAPF.